roles/openshift_logging/templates/fluentd.j2

```diff
@@ -63,27 +63,27 @@ spec:
         - name: "K8S_HOST_URL"
           value: "{{master_url}}"
         - name: "ES_HOST"
-          value: "{{es_host}}"
+          value: "{{openshift_logging_es_host}}"
         - name: "ES_PORT"
-          value: "{{es_port}}"
+          value: "{{openshift_logging_es_port}}"
         - name: "ES_CLIENT_CERT"
-          value: "{{es_client_cert}}"
+          value: "{{openshift_logging_es_client_cert}}"
         - name: "ES_CLIENT_KEY"
-          value: "{{es_client_key}}"
+          value: "{{openshift_logging_es_client_key}}"
         - name: "ES_CA"
-          value: "{{es_ca}}"
+          value: "{{openshift_logging_es_ca}}"
         - name: "OPS_HOST"
           value: "{{ops_host}}"
         - name: "OPS_PORT"
-          value: "{{ops_port}}"
+          value: "{{openshift_logging_es_ops_port}}"
```

Comment:
@ewolinetz i have questions regarding these ops vars.  The calling tasks sets ops_host  but none of the others which it seems like it should